DotNet/ExternalTask/Program.cs

```diff
@@ -25,7 +25,7 @@ private static async Task RunSampleExternalTaskWorker()
 
             Console.WriteLine("Warten auf Aufgaben für das Topic 'AktivierungsemailSenden'.");
 
-            await client.WaitForHandle<TestPayload>("AktivierungsemailSenden", async (externalTask) => {
+            await client.SubscribeExternalTask<TestPayload>("AktivierungsemailSenden", async (externalTask) => {

```

Comment:
Find ich gut, :+1: 

Würd ich evtl. noch etwas expliziter machen.

```suggestion
            await client.SubscribeToExternalTaskTopic<TestPayload>("AktivierungsemailSenden", async (externalTask) => {
```

Reply:
@ElRaptorus ja, ich subscribe mich auf einen Topic bekomme im `HandleExternalTaskAction ` aber einen ExternalTask zur Verfügung gestellt. Meinst Du nicht, dass dann SubscribeExternalTask besser wäre?

Reply:
Ja, aber die gehören ja immer nur zu einem bestimmten Topic.

Wie wärs mit `SubscribeToExternalTasksWithTopic`?

Reply:
Ich fänd aber auch `SubscribeToExternalTask` ok. :+1: